generator/ServiceClientGeneratorLib/UnitTestProjectFileCreator.cs

```diff
@@ -96,6 +96,11 @@ public void Execute(string unitTestRoot, IEnumerable<ServiceConfiguration> servi
                     }
                     projectProperties.FxcopAnalyzerRuleSetFilePath = Utils.PathCombineAlt("..", "..", "..", "..", "AWSDotNetSDK.ruleset");
                     projectProperties.FxcopAnalyzerRuleSetFilePathForBuild = Utils.PathCombineAlt("..", "..", "..", "..", "AWSDotNetSDKForBuild.ruleset");
+                    
+                    if (_serviceName == "DynamoDBv2")
```

Comment:
It doesn't hurt to just sign all unit test projects. I would suggest remove the if condition and always setting `SignBinaries` to `true`.